Makefile

```diff
@@ -398,6 +398,18 @@ endef
 		make all install; \
 	fi
 
+
+HOOK_BIN_DIR ?= /usr/libexec/oci/hooks.d/
+HOOK_DIR ?= /usr/share/containers/oci/hooks.d
+
+generate-seccomp.install: 
```

Comment:
Other tnstall targets are prefixed with `install.`. Also this should be added as a dependency to the main `install` target.

Reply:
Let's rename it to `install.oci-trace-hook`.